salt/modules/mac_package.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+'''
+Install pkg, dmg and .app applications on Mac OS X minions.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import os
+import logging
+
+# Import salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+__virtualname__ = "macpackage"
+
+
+def __virtual__():
+    '''
+    Only work on Mac OS
+    '''
+    if salt.utils.is_darwin():
+        return __virtualname__
+    return False
+
+
+def install(pkg, target="LocalSystem", store=False, allow_untrusted=False):
+    '''
+    Install a pkg file
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' macpackage.install test.pkg
+
+    target
+        The target in which to install the package to
+
+    store
+        Should the package be installed as if it was from the store?
+
+    allow_untrusted
+        Allow the installation of untrusted packages?
+
+
+    '''
+    cmd = 'installer -pkg {0} -target {1}'.format(pkg, target)
+    if store:
+        cmd += ' -store'
+    if allow_untrusted:
+        cmd += ' -allowUntrusted'
+
+    # We can only use wildcards in python_shell which is
+    # sent by the macpackage state
+    python_shell = False
+    if '*.' in cmd:
+        python_shell = True
```

Comment:
If we do this, then we have to ensure that the command that's being sent is safe. Otherwise, it's trivial to inject an additional command in here and have the shell execute it.